extensions/sshkeys/downloadsshkeys.go

```diff
@@ -18,7 +19,7 @@ const (
 // the ssh key for a particular node.
 func DownloadSSHKeys(client *rancher.Client, machinePoolNodeName string) ([]byte, error) {
 	machinePoolNodeNameName := fmt.Sprintf("fleet-default/%s", machinePoolNodeName)
-	machine, err := client.Steve.SteveType(ClusterMachineConstraintResourceSteveType).ByID(machinePoolNodeNameName)
+	machine, err := client.Steve.SteveType(stevetypes.Machine).ByID(machinePoolNodeNameName)
```

Comment:
just double checking, this was the same thing as `ClusterMachineConstraintResourceSteveType`?

Reply:
I just double checked, yes they are the same.